### PR TITLE
CMP-2924: Add support for url_passthrough, ads_data_redaction params.…

### PR DESCRIPTION
**Pull request by EdgardoRodriguezSolano**
This PR 
- Adds support for both `url_passthrough` and `ads_data_redaction` params
- Allows to specify the GCM default settings on a region basis
- Adds tests covering these cases
- 
The new UI looks like this
Screenshot 2022-09-20 at 11 24 07

### CODE REVIEW

File: template.tpl [hunk 1]
```diff
@@ -19,8 +19,8 @@ ___INFO___
     "PERSONALIZATION"
   ],
   "brand": {
-    "id": "Didomi",
-    "displayName": "Didomi",
+    "id": "github.com_didomi",
```
Comment (jawadst) [inline template.tpl:22]: Why this change?

Comment (EdgardoRodriguezSolano) [reply template.tpl:22]: That one got initially changed here https://github.com/didomi/gtm-template/commit/57c1f50f2ada3077859102ec1c526e8cd5affd05

It comes directly from the template
Screenshot 2022-09-20 at 19 30 32

I have been manually keeping the pair "id/displayName". Leaving it like this should help us to just copy/paste the TPL file export and not modify this field.

**Review by jawadst — APPROVED** [review]
